src/DependencyInjection/SonataFormatterExtension.php

```diff
@@ -106,7 +106,7 @@ public function getAlias(): string
 
     private function createEnvironment(ContainerBuilder $container, string $code, array $extensions): string
     {
-        $loader = new Definition('Twig_Loader_Array');
+        $loader = new Definition('\Twig\Loader\ArrayLoader');
```

Comment:
can you import the classes and do `ArrayLoader::class`?

Reply:
Yes, of course! Done!